scripts/create_kiwix-desktop_appImage.sh

```diff
@@ -43,5 +78,33 @@ chmod u+x linuxdeployqt
 ./linuxdeployqt $APPDIR/usr/bin/kiwix-desktop -bundle-non-qt-libs -extra-plugins=imageformats,iconengines
 # Fix the RPATH of QtWebEngineProcess [TODO] Fill a issue ?
 patchelf --set-rpath '$ORIGIN/../lib' $APPDIR/usr/libexec/QtWebEngineProcess
-# Build the image.
-./linuxdeployqt $APPDIR/usr/share/applications/kiwix-desktop.desktop -bundle-non-qt-libs -extra-plugins=imageformats,iconengines -appimage
+
+./linuxdeployqt $APPDIR/usr/share/applications/kiwix-desktop.desktop -bundle-non-qt-libs -extra-plugins=imageformats,iconengines
```

Comment:
I'm not sure this line is necessary now.
`$APPDIR` should have been prepared by previous `liunxdeployqt` (and all other cp/patch)
This line (especially the `-appimage` option) was just to actually create the appimage.

But if you use `appimagetool`, this should not be necessary.

Reply:
The difference of this line from the first invocation of `linuxdeployqt` is that it receives as input the desktop file (`kiwix-desktop.desktop`) instead of the binary executable. I concluded that the result is that it embeds the icon into the appimage (but didn't investigate whether it is actually so).

Reply:
I replaced this line by a couple of `cp` commands.

Reply:
And fixed the CI too.